acceptance/new-user/sign-up.test.js

```diff
@@ -1,105 +1,115 @@
-const puppeteer = require('puppeteer');
-const config = require('config');
+const puppeteer = require("puppeteer");
+const config = require("config");
 
-const URLS = require('../../constants/urls');
+const URLS = require("../../constants/urls");
 
 const { HOME_PAGE, SIGN_UP_PAGE, MY_HOST } = URLS;
 
 let browser, page;
 
+function delay(time) {
+    return new Promise(function (res) {
+        setTimeout(res, time);
+    });
+}
+
 beforeAll(async () => {
-  browser = await puppeteer.launch({
-    // headless: false,
-    // slowMo: 250
-  });
-
-  const context = await browser.createIncognitoBrowserContext();
-
-  page = await context.newPage();
-  // await page.emulate(puppeteer.devices['iPhone 8']);
-  // await page.setViewport({
-  //   width: 1440,
-  //   height: 900,
-  //   deviceScaleFactor: 1,
-  // });
-  return page;
+    browser = await puppeteer.launch({
+        headless: false,
+        slowMo: 0,
+    });
+
+    const context = await browser.createIncognitoBrowserContext();
+
+    page = await context.newPage();
+    // await page.emulate(puppeteer.devices['iPhone 8']);
+    // await page.setViewport({
+    //   width: 1440,
+    //   height: 900,
+    //   deviceScaleFactor: 1,
+    // });
+    return page;
 });
 
 afterAll(async () => {
-  await browser.close();
+    await browser.close();
 });
 
+describe("New user navigates", () => {
+    test("New user sess a sign up page", async () => {
+        await page.goto(HOME_PAGE);
+
+        await page.waitForResponse((res) => res.url().endsWith("/users/self"));
+        await page.waitForSelector("button.login-btn-text");
+
+        await Promise.all([
+            page.waitForNavigation(),
+            page.click("button.login-btn-text"),
+        ]);
+
+        const ghUsernameInput = await page.waitForSelector("input#login_field");
+        const ghPasswordInput = await page.waitForSelector("input#password");
+
+        await ghUsernameInput.type(config.get("testUser.username"));
+        await ghPasswordInput.type(config.get("testUser.password"));
+        await Promise.all([
+            page.waitForNavigation(),
+            page.keyboard.press("Enter"),
+        ]);
+        await page.waitForSelector("button#js-oauth-authorize-btn");
```

Comment:
re: naming convention !

Reply:
Again, this selector is for the button in the authorize page of github which will prompt the new user to authorize Real Dev Squad when they first sign in. And I think this particular page is controlled by github so we have to go with their naming convention. Although now that you've brought it up, we'll have to think about some way around this, because if github changes their naming convention or class name.. our tests could break 😅

Reply:
OK 
You have a point in your last line explanation 🧠